Automattic/jetpack
# Social: Add the change setting logic to the auto-conversion feature

My previous change added the notice for the auto-conversion, and disabled the connections accordingly. This change adds the functionality to the `Change setting` button, which opens the Jetpack Social admin page, or the Jetpack settings sharing section for Jetpack in a new tab, then refreshes the state on focus

## Proposed changes:

* 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Turn off Social and have Jetpack enabled.
  * Go to the Jetpack settings page and turn on the auto conversion setting
  * Open up the editor and create a new post
  * Select a media file that is convertible, but not valid for some connections - for example a 10Mb jpg image
  * You should see the notice that it will be converted. If you dismissed already, run `jetpack docker wp option delete jetpack_social_dismissed_notices` to bring it back
  * On the notice click change settings button. It should open up Jetpack settings on the sharing screen.
  * Turn off the auto conversion.
  * Go back to the editor, the page should reflect the changes without needing to refresh

* Do the same with Jetpack Social enabled only. The only difference is that the button should direct you to the social admin page

<img width="282" alt="CleanShot 2023-09-01 at 18 03 14 png" src="https://github.com/Automattic/jetpack/assets/36671565/29c3e267-5fb6-47be-b873-c11abcab6fcc">
